src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/CaseActionsForCaseWorkerControllerTest.java

```diff
@@ -1649,4 +1651,36 @@ public void initialiseAmendCaseDetailsForbidden() throws Exception {
         verify(clerkService, never()).initialiseClerkResponsible(isA(CaseData.class));
         verify(fileLocationSelectionService, never()).initialiseFileLocation(isA(CaseData.class));
     }
+
+    @Test
+    public void addServiceIdUrl_tokenOk() throws Exception {
```

Comment:
can we verify setHmctsServiceIdSupplementary() is called in the controller method?